twim/calibration/R3BTwimMapped2Cal.cxx

```diff
@@ -244,6 +244,9 @@ void R3BTwimMapped2Cal::Exec(Option_t* option)
                 Int_t ii = fNumPosParams * i;
                 Float_t a0 = PosParams[s]->GetAt(ii);
                 Float_t a1 = PosParams[s]->GetAt(ii + 1);
+                Float_t a2 = PosParams[s]->GetAt(ii + 2);
```

Comment:
Is it possible to add a criterion as below? This is just a compatibility purpose for people who are using only 2 parameters for each channel.
`Float_t a2 = 0.0;`
` if(fNumPosParams>2) a2 = PosParams[s]->GetAt(ii + 2);`